app/presenters/workflow_presenter.rb

```diff
@@ -3,9 +3,11 @@
 class WorkflowPresenter
   # @param [Object] view the rails view context
   # @param [WorkflowStatus] workflow_status
-  def initialize(view:, workflow_status:)
+  # @param [Cocina::Models::DRO] item the repository object that the workflow is about
+  def initialize(view:, workflow_status:, item:)
```

Comment:
Should `item` be `cocina_item` or `cocina_object` instead?

Reply:
@jcoyne here's the one that's outstanding

Reply:
Thank you. I've got it.